ganga/GangaCore/Utility/Virtualization.py

```diff
@@ -8,6 +8,21 @@
 import urllib
 import os
 
+def checkNative():
```

Comment:
Move this to a separate file as it is misleading to have inside virtualization.py